docs/tracing.md

```diff
@@ -41,8 +40,8 @@ In Kubernetes it is as easy as (on Thanos sidecar example):
 
 1. Create new directory under `pkg/tracing/<provider>`
 2. Implement `opentracing.Tracer` interface
-3. Add client implementation to the factory in [factory](/pkg/tracing/client/factory.go) code. (Using as small amount of flags as possible in every command)
-4. Add client struct config to [cfggen](/scripts/cfggen/main.go) to allow config auto generation.
+3. Add client implementation to the factory in [factory](../pkg/tracing/client/factory.go) code. (Using as small amount of flags as possible in every command)
```

Comment:
Let's add feature to mdox transform that will glue github prefix (with correct version to out of docs links)